src/object/srv_internal.h

```diff
@@ -30,8 +30,9 @@ struct migrate_pool_tls {
 	/* POOL UUID and pool to be migrated */
 	uuid_t			mpt_pool_uuid;
 	struct ds_pool_child	*mpt_pool;
-	uint64_t		mpt_global_version;
 	unsigned int		mpt_version;
+	uint32_t		mpt_global_version;
+	uint32_t		mpt_layout_version;
```

Comment:
a few comment for the global_ver and layout_ver will be helpful, thx

Reply:
sure.